addons/website_event_questions/i18n/website_event_questions.pot

```diff
@@ -227,7 +227,7 @@ msgstr ""
 #. module: website_event_questions
 #: code:addons/website_event_questions/models/event_question.py:0
 #, python-format
-msgid "Question cannot belong to both the event category and itself."
+msgid "Question cannot be linked to both an Event and an Event Type."
```

Comment:
Ok to change the error message which is not really clear; however don't remove the old string, just add the new one. That way people using the old code can effectively benefit from translations.